tests/Functional/ContentEventRenderTest.php

```diff
@@ -132,4 +135,67 @@ public function testEventFeaturedMediaTranslation(): void {
     }
   }
 
+  /**
+   * Test registration button description.
+   */
+  public function testEventRegistrationDateDescription(): void {
+    // Freeze the time at a specific point.
+    $static_time = new DrupalDateTime('2020-02-01 16:00:00', DateTimeItemInterface::STORAGE_TIMEZONE);
+    $start_date = (clone $static_time)->modify('+15 days');
+    $end_date = (clone $static_time)->modify('+20 days');
+
+    // Registration will be started today.
+    $registration_start_date = (clone $static_time)->modify('+5 hours');
+    $registration_end_date = (clone $static_time)->modify('+10 days');
+
+    $time = \Drupal::time();
+    $time->freezeTime();
+    $time->setTime($static_time->getTimestamp());
+
+    // Create a Event node.
+    $node = $this->nodeStorage->create([
+      'type' => 'oe_event',
+      'title' => 'Test event node',
+      'oe_event_type' => 'http://publications.europa.eu/resource/authority/public-event-type/COMPETITION_AWARD_CEREMONY',
+      'oe_teaser' => 'Event teaser',
+      'oe_subject' => 'http://data.europa.eu/uxp/1000',
+      'oe_event_status' => 'as_planned',
+      'oe_event_dates' => [
+        'value' => $start_date->format(DateTimeItemInterface::DATETIME_STORAGE_FORMAT),
+        'end_value' => $end_date->format(DateTimeItemInterface::DATETIME_STORAGE_FORMAT),
+      ],
+      'oe_event_languages' => [
+        ['target_id' => 'http://publications.europa.eu/resource/authority/language/EST'],
+        ['target_id' => 'http://publications.europa.eu/resource/authority/language/FRA'],
+      ],
+      'oe_event_registration_dates' => [
+        'value' => $registration_start_date->format(DateTimeItemInterface::DATETIME_STORAGE_FORMAT),
+        'end_value' => $registration_end_date->format(DateTimeItemInterface::DATETIME_STORAGE_FORMAT),
+      ],
+      'oe_event_entrance_fee' => 'fee',
+      'oe_event_registration_capacity' => 'capacity',
+      'oe_event_registration_url' => 'http://www.example.com/registation',
+      'oe_author' => 'http://publications.europa.eu/resource/authority/corporate-body/ACJHR',
+      'oe_content_content_owner' => 'http://publications.europa.eu/resource/authority/corporate-body/COMMU',
+      'uid' => 0,
+      'status' => 1,
+    ]);
+    $node->save();
+    $this->drupalGet($node->toUrl());
+
+    // Assert registration date description when registration will start today.
+    $registration_info_content = $this->assertSession()->elementExists('css', 'p.ecl-u-type-paragraph.ecl-u-type-color-grey-75');
+    $this->assertEquals('Registration will open today, 2 February 2020, 08:00.', $registration_info_content->getText());
+
+    // Assert registration date description when registration will end today.
+    $registration_start_date = (clone $static_time)->modify('-10 days');
+    $registration_end_date = (clone $static_time)->modify('+1 hours');
+    $node->set('oe_event_registration_dates', [
```

Comment:
The event itself has one set of date values but the users would be accessing it from different timezones so updating only the date instead of the timezone just to make sure it works in Sydney timezone doesn't seem to relevant. We could test the same set of date values in different timezones to make sure the issue is fixed.